.github/workflows/commit-workflow-all.yml

```diff
@@ -0,0 +1,369 @@
+name: commit-workflow-allinone
+
+on:
+  push:
+    branches: [ main, continuous_integration ]
+  workflow_dispatch:
+    branches: [ main, continuous_integration ]
+  pull_request:
+    branches: [ main, continuous_integration ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+
+  test-env:
```

Comment:
Do we need this step? If yes, I believe the step name Whatever` can be improved ;) Also `Oh man` in the line 22 :D